docs/get_started/get_started.md

```diff
@@ -50,26 +50,12 @@ admission controller delegating all scheduling to YuniKorn. Because this mode is
 
 If you are unsure which deployment mode you should use, refer to our [side-by-side comparison](user_guide/deployment_modes).
  
-If you want to use specific YuniKorn version, you can update helm chart via helm upgrade.
+If you want to use a specific version of YuniKorn, you can add the `--version` flag to the `helm install` command as follows:
 ```shell script
-helm upgrade -f custom.yml  -install yunikorn yunikorn/yunikorn -n yunikorn --create-namespace
+helm install yunikorn yunikorn/yunikorn --namespace yunikorn --version {version}
 ```
 
-The custom.yml is
-```yml
-image:
-  tag: scheduler-{version}    # default depends on YuniKorn version
-
-admissionController:
-  image:
-    tag: admission-{version}  # default depends on YuniKorn version
-
-web:
-  image:
-    tag: web-{version}        # default depends on YuniKorn version
-```
-
-You can check available `{version}` on [DockerHub](https://hub.docker.com/r/apache/yunikorn/tags).
+You can check available `{version}` on [here](https://artifacthub.io/packages/helm/yunikorn/yunikorn).
```

Comment:
nit: I'm not in favor of highlighting `{version}` here.  Could we remove the highlight? 

The description for Docker version is relatively simple and clear.

<img width="531" alt="image" src="https://github.com/user-attachments/assets/e70c6046-7008-4910-a8ef-11e856445545">